test/xds/xds_server_integration_test.go

```diff
@@ -50,6 +51,15 @@ func (*testService) UnaryCall(context.Context, *testpb.SimpleRequest) (*testpb.S
 	return &testpb.SimpleResponse{}, nil
 }
 
+func (*testService) FullDuplexCall(stream testgrpc.TestService_FullDuplexCallServer) error {
```

Comment:
This `testService` type is a complete no-op test service implementation. This could be replaced with a call to `stubserver.StartTestService` in most cases where the streaming RPC is not required. And in the odd case where the streaming RPC is required, the test could instantiate a stubserver and fill in this implementation for `FullDuplexCall`.

Reply:
Sorry, I don't get what you're saying here.